lib/policyEvaluator/RequestContext.ts

```diff
@@ -18,6 +18,8 @@ import {
 const _actionNeedQuotaCheck = {
     objectPut: true,
     objectPutPart: true,
+    objectPutVersion : true,
+    objectRestore: true,
```

Comment:
I will do that in a dedicated PR (with you as the author), as it requires some other changes in the code base

Reply:
Made it on a separate commit to be able to drop it => done